src/hook.js

```diff
@@ -59,7 +59,7 @@ process.on('message', msg => {
 });
 
 process.on('uncaughtException', err => {
-  const name = electron.app.getName();
+  const { name } = electron.app;
```

Comment:
It seems that this property did not always exist. I went back to some older documentation and it was not there. Since electronmon supports multiple versions of Electron, I don't think that we can just assume no one will actually use older versions. We should probably check what exists and use either `name` or `getName()` here.

Reply:
Yep, good catch! Just looked back for reference and it seems like it only made its way into Electron v7.0.0 stable (electron/electron#17701).